test/unit/test_logger.py

```diff
@@ -126,3 +126,33 @@ def test_cyan_text():
     x = '{}{}{}'.format(colorama.Fore.CYAN, 'foo', colorama.Style.RESET_ALL)
 
     assert x == logger.cyan_text('foo')
+
+
+def test_markup_detection_pycolors0(monkeypatch):
+    monkeypatch.setenv('PY_COLORS', '0')
+    assert not logger.should_do_markup()
+
+
+def test_markup_detection_pycolors1(monkeypatch):
+    monkeypatch.setenv('PY_COLORS', '1')
+    assert logger.should_do_markup()
+
+
+class FileHandleTTY(object):
```

Comment:
put
```python
__metaclass__ = type
```
in the beginning of the module and you won't have to inherit from object explicitly.
Oh, and since you don't use features of new-style classes you don't need this anyway...